sig-windows/README.md

```diff
@@ -92,4 +92,16 @@ As always, you can also find the active list of open issues and proposals [Open
 
 Past meetings are in the [Recorded Meetings Playlist on Youtube](https://www.youtube.com/playlist?list=PL69nYSiGNLP2OH9InCcNkWNu2bl-gmIU4&jct=LZ9EIvD4DGrhr2h4r0ItaBmco7gTgw)
 
+## Sig-Windows is Growing! 
+
+Sig-Windows is looking to build new commiters and owners, with the goal of eventually having new leaders and chairs.
+
+If you are interested in this path, 
+
+- *First*, learn how to install, build, and run upstream Kubernetes bits with windows.  This might take a week or two as it is non-trivial, but understanding the upstream componentry, and contributing to it, is the first step to taking on a role as a new sig-windows contributor. and improving our documentation for others.
```

Comment:
```suggestion
- *First*, learn how to install, build, and run upstream Kubernetes bits with windows.  This might take a week or two as it is non-trivial, but understanding the upstream componentry, and contributing to it, is the first step to taking on a role as a new sig-windows contributor and improving our documentation for others.
```